packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -210,6 +211,18 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// If null then [kMaxFlingVelocity] is used.
   double? maxFlingVelocity;
 
+  /// Whether the drag callbacks should only be dispatched when the drag threshold
+  /// has been met.
+  ///
+  /// If true, the drag callbacks will only be dispatched when this recognizer has
+  /// won the arena and the drag threshold has been met.
+  ///
+  /// If false, the drag callbacks will be dispatched when this recognizer has won
+  /// the arena.
+  ///
+  /// This value defaults to false.
+  bool onlyDispatchDragCallbacksWhenThresholdMet;
```

Comment:
Nit: What if this was something like `double threshold = gestureSettings?.touchSlop;`?

Or if that's not possible, maybe shorten the name to "useThreshold" or something?

I'm just throwing out ideas, feel free to ignore all of this :)

Reply:
I don't think this is possible. There is already a threshold being checked in `_hasSufficientGlobalDistanceToAccept` in `handleEvent`, the issue is that if the recognizer is in the arena by itself it gets accepted by default. In the case of bottom sheet this happens pretty much immediately on the first `PointerDownEvent`/ tap. `acceptGesture` is then called which fires the `drag*` callbacks. The threshold check never happens before acceptance. 

I do agree though this needs a better name.

maybe `onlyAcceptDragOnThreshold`?